package.json

```
@@ -27,6 +27,7 @@
   "devDependencies": {
     "@commitlint/cli": "16.2.3",
     "@commitlint/config-angular": "16.2.3",
+    "@fastify/swagger": "^6.0.0",
```

what about this one? I'm pretty sure that renovate bot will pin this too

```suggestion
    "@fastify/swagger": "6.0.0",
```